src/index.ts

```diff
@@ -1,8 +1,10 @@
 import { DebugHandler, DebugConsume } from "./debugging";
 import * as en from "./locales/en";
 import { Chrono, Parser, Refiner } from "./chrono";
+import { TimezoneAbbrMap } from "./timezone";
+import type { ParsingResult } from "./results";
 
-export { en, Chrono, Parser, Refiner };
+export { en, Chrono, Parser, Refiner, TimezoneAbbrMap, ParsingResult };
```

Comment:
Please see the newest changes. The changes are mostly according to your suggestion, and there's now better documentation :)

However, I kept the name `AmbiguousTimezoneMap` because I feel like it better describes the uncertainty associated with parsing these kinds of timezones. Moreover, I feel like "WithDstInfo" may incorrectly give the impression that these timezones are like normal timezone mappings, but with additional information. On the contrary, the use of these mappings reduces the certainty of the parsing due to the DST ambiguity.